main.go

```diff
@@ -247,6 +247,7 @@ func (s *server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		copy(blockHash[:], blockHashBytes)
 		numByHash := s.eth1BlockNumbersByHash[blockHash]
 		block := s.eth1BlocksByNumber[numByHash]
+		block.Time = uint64(time.Now().Unix())
```

Comment:
this breaks if we have multiple nodes, we end up having different genesis time for different nodes

Reply:
Agreed, will resolve